src/elements/dropdownList/dropdownList.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "normalize.css";
+@import "sanitize.css";
+
+.list{
+    display: none;
+    z-index: 9999999;
```

Comment:
why so much? best practice to use numbers between 0...100

Reply:
all z-indexes in the project are changed to 9 or 99, where 9 was not enough